truffle/test/Splitter.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const Splitter = artifacts.require('Splitter');
+const truffleAssert = require('truffle-assertions');
+
+contract('Splitter', (accounts) => {
+    let instance;
+
+    // Initiating new contract before each test
+    beforeEach("Creating new contract", async () => {
+        await Splitter.new()
+            .then(_instance => {
+                instance = _instance;
+            });
+    });
+
+    //Testing the balance update
+    it('Balance should be equal to 10 ether', () => {
+        return instance.splitEth(accounts[1], accounts[2], {from: accounts[0], value: web3.utils.toWei("10", "ether")})
+            .then(txObj => {
+                assert.strictEqual(txObj.logs.length, 1);
+                assert.strictEqual(txObj.logs[0].event, "LogEthSent");
+                return web3.eth.getBalance(instance.address);
+            })
+            .then(balance => {
+                assert.strictEqual(balance, web3.utils.toWei("10", "ether"));
+            });
+    });
+
+    // Testing to split ether if the contract has been paused
+    it('The splitEth function should not be usable when the contract is paused', () => {
+        return instance.pauseContract()
+            .then(async txObj => {
+                assert.strictEqual(txObj.logs.length, 1);
+                assert.strictEqual(txObj.logs[0].event, "LogPausedContract");
```

Comment:
These asserts belong in a `pausable.js`.

Reply:
It is already in the ```Stoppable.js``` test file. However, I wanted to verified all use cases for each contract